relayer/provider/provider.go

```diff
@@ -39,7 +40,7 @@ type ChainProvider interface {
 	SetAdmin(context.Context, string) error
 
 	FinalityBlock(ctx context.Context) uint64
-	GenerateMessages(ctx context.Context, messageKey *types.MessageKeyWithMessageHeight) ([]*types.Message, error)
+	GenerateMessages(ctx context.Context, fromHeight, toHeight uint64) ([]*types.Message, error)
```

Comment:
Should we inform or document somewhere before chaing the behavior of existing command and replacing with the new one ?? any reasons for this change as we move from specific relay message to range which has its own downsides.

Reply:
It does not change the inner working. messageKey struct input is somewhat redundant.